<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenProxifier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>15</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="proxyGroup">
      <property name="title">
       <string>SOCKS5 Proxy Settings</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="historyLabel">
         <property name="text">
          <string>History:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="historyLayout">
         <item>
          <widget class="QComboBox" name="serverHistoryCombo">
           <property name="minimumWidth">
            <number>200</number>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveServerButton">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="maximumWidth">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteServerButton">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="maximumWidth">
            <number>60</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="hostLabel">
         <property name="text">
          <string>Server:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="hostLayout">
         <item>
          <widget class="QLineEdit" name="proxyHostEdit">
           <property name="placeholderText">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="proxyPortSpin">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>1080</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="authCheckBox">
         <property name="text">
          <string>Require Authentication</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="userLabel">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="usernameEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="passLabel">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="passwordEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout" name="testLayout">
         <item>
          <widget class="QPushButton" name="testServerButton">
           <property name="text">
            <string>Test Connection</string>
           </property>
           <property name="minimumHeight">
            <number>30</number>
           </property>
           <property name="styleSheet">
            <string>QPushButton { background-color: #2196F3; color: white; font-weight: bold; }
QPushButton:hover { background-color: #1976D2; }
QPushButton:disabled { background-color: #cccccc; }</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="connectionStatusLabel">
           <property name="text">
            <string></string>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="targetGroup">
      <property name="title">
       <string>Target Processes (Auto-Monitor)</string>
      </property>
      <layout class="QVBoxLayout" name="targetLayout">
       <item>
        <layout class="QHBoxLayout" name="addExeLayout">
         <item>
          <widget class="QLineEdit" name="exeNameEdit">
           <property name="placeholderText">
            <string>Enter exe name (e.g., Antigravity.exe)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addExeButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeExeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="exeListWidget">
         <property name="minimumHeight">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoStartCheckBox">
         <property name="text">
          <string>Auto-start monitoring on launch</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="monitorLayout">
         <item>
          <widget class="QPushButton" name="startMonitorButton">
           <property name="text">
            <string>Start Monitoring</string>
           </property>
           <property name="minimumHeight">
            <number>35</number>
           </property>
           <property name="toolTip">
            <string>Monitor system for target processes and auto-inject</string>
           </property>
           <property name="styleSheet">
            <string>QPushButton { background-color: #4CAF50; color: white; font-weight: bold; }
QPushButton:hover { background-color: #45a049; }
QPushButton:disabled { background-color: #cccccc; }</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopMonitorButton">
           <property name="text">
            <string>Stop Monitoring</string>
           </property>
           <property name="minimumHeight">
            <number>35</number>
           </property>
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string>QPushButton { background-color: #f44336; color: white; font-weight: bold; }
QPushButton:hover { background-color: #da190b; }
QPushButton:disabled { background-color: #cccccc; }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="logGroup">
      <property name="title">
       <string>Activity Log</string>
      </property>
      <layout class="QVBoxLayout" name="logLayout">
       <item>
        <widget class="QTextEdit" name="logTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="minimumHeight">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="statusLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status: Ready</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="languageCombo">
        <property name="minimumWidth">
         <number>80</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
